<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>component</class>
 <widget class="QWidget" name="component">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Clock Component</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_">
     <item>
      <widget class="QPushButton" name="componentButton">
       <property name="text">
        <string>Select Component...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="optionsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Options...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="previewLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
